code/modules/surgery/implant.dm

```diff
@@ -175,6 +175,12 @@
 		to_chat(user, "<span class='warning'>Central command would kill you if you implanted the disk into someone.</span>")
 		return 0//fail
 
+	if(istype(tool, /obj/item/weapon/storage/box))
+		for(var/obj/item/I in tool.contents)
```

Comment:
eh, correct me if i'm wrong but locate() seems to do what i want already, I'd still need to iterate through what GetAllContents() would return